deployment/network-operator/README.md

```diff
@@ -216,6 +216,20 @@ Specifies components to deploy in order to facilitate a secondary network in Kub
 | `ipamPlugin.repository` | string | `dougbtv` | IPAM CNI Plugin image repository  |
 | `ipamPlugin.version` | string | `v0.3` | IPAM CNI Plugin image version  |
 
+## Helm Tests
```

Comment:
i would prefer to put it under the quickstart section and not all the way at the end.